modules/flippBidAdapter.md

```diff
@@ -32,9 +32,10 @@ var adUnits = [
                     publisherNameIdentifier: 'wishabi-test-publisher', // Required
                     siteId: 1192075, // Required
                     zoneIds: [260678], // Optional
-                    userKey: "", // Optional
+                    userKey: `4188d8a3-22d1-49cb-8624-8838a22562bd`, // Optional
```

Comment:
Maybe we should add to the comment that this example was generated by uuid?

Reply:
That makes sense, just updated the branch